backend/src/main/java/com/imsnacks/Nyeoreumnagi/work/service/MyWorkService.java

```diff
@@ -7,7 +7,7 @@
 import com.imsnacks.Nyeoreumnagi.work.dto.request.ModifyMyWorkRequest;
 import com.imsnacks.Nyeoreumnagi.work.dto.request.RegisterMyWorkRequest;
 import com.imsnacks.Nyeoreumnagi.work.dto.response.ModifyMyWorkResponse;
-import com.imsnacks.Nyeoreumnagi.work.dto.response.ResisterMyWorkResponse;
+import com.imsnacks.Nyeoreumnagi.work.dto.response.ResgisterMyWorkResponse;
```

Comment:
P1: 또 다른 오타가 생겼습니다!!

```suggestion
import com.imsnacks.Nyeoreumnagi.work.dto.response.RegisterMyWorkResponse;
```